packages/charts/src/components/ColumnChartWithTrend/ColumnChartWithTrend.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+import { DocsHeader } from '@shared/stories/DocsHeader';
+import '@ui5/webcomponents-icons/dist/person-placeholder.js';
+import { ColumnWithTrendChart } from '@ui5/webcomponents-react-charts/dist/ColumnWithTrendChart';
+import { complexDataSet } from '../../resources/DemoProps';
+
+<Meta
+  title="Charts /  ColumnChart with Trend"
+  component={ColumnWithTrendChart}
+  argTypes={{
+    dataset: {
+      control: { disable: true }
+    },
+    children: {
+      control: { disable: true }
+    }
+  }}
+/>
+
+<DocsHeader />
+
+## Example
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      dataset: complexDataSet,
+      style: { height: '60vh' },
+      dimensions: [
+        {
+          accessor: 'name',
+          formatter: (d) => `${d} 2019`
+        }
+      ],
+      measures: [
+        {
+          accessor: 'users',
+          label: 'Users',
+          formatter: (val) => val.toLocaleString(),
+          type: 'line'
+        },
+        {
+          accessor: 'sessions',
+          label: 'Active Sessions',
+          type: 'column'
+        }
+      ]
+    }}
```

Comment:
Please move default `args` to `Meta` component, and only overwrite story-specific `args` on the `Story` itself. This saves a lot of time when creating or changing stories :)